---
title: "TCR sequencing paired with massively parallel 3′ RNA-seq reveals clonotypic T cell signatures"
collection: publications
permalink: https://www.nature.com/articles/s41590-019-0544-5
excerpt: 
date: 2019-11-19
venue: 'Nature Immunology'
paperurl: 'https://doi.org/10.1038/s41590-019-0544-5'
citation: 'Tu, A. A., Gierahn, T. M., Monian, B., <b>Morgan, D. M.</b>, Mehta, N. K., Ruiter, B., Shreffler, W. G., Shalek, A. K., Love, J. C. (2019). &quot;TCR sequencing paired with massively parallel 3′ RNA-seq reveals clonotypic T cell signatures.&quot; <i>Nature Immunology</i>.<b> 20 </b>1692-1699 (2019).'
---
High-throughput 3′ single-cell RNA-sequencing (scRNA-seq) allows cost-effective, detailed characterization of individual immune cells from tissues. Current techniques, however, are limited in their ability to elucidate essential immune cell features, including variable sequences of T cell antigen receptors (TCRs) that confer antigen specificity. Here, we present a strategy that enables simultaneous analysis of TCR sequences and corresponding full transcriptomes from 3′-barcoded scRNA-seq samples. This approach is compatible with common 3′ scRNA-seq methods, and adaptable to processed samples post hoc. We applied the technique to identify transcriptional signatures associated with T cells sharing common TCRs from immunized mice and from patients with food allergy. We observed preferential phenotypes among subsets of expanded clonotypes, including type 2 helper CD4+ T cell (TH2) states associated with food allergy. These results demonstrate the utility of our method when studying diseases in which clonotype-driven responses are critical to understanding the underlying biology.

[Download paper here](http://duncanmorgan.github.io/files/NatImmunol2019.pdf)

