---
title: "WAT3R: Recovery of T-Cell Receptor Variable Regions from 3` Single-Cell RNA Sequencing."
collection: publications
permalink: https://doi.org/10.1101/2022.01.26.477886
excerpt: 
date: 2022-01-28
venue: 'bioRxiv'
paperurl: 'https://doi.org/10.1101/2022.01.26.477886'
citation: 'Ainciburu, M., <b>Morgan, D. M.<\b>, DePasquale, E. A. K., Love, J., C., Propser, F., van Galen, P. &quot;WAT3R: Recovery of T-Cell Receptor Variable Regions from 3` Single-Cell RNA Sequencing.&quot; <i>bioRxiv</i>.
---
High-throughput 3′ single-cell RNA-sequencing (scRNA-seq) allows cost-effective, detailed characterization of individual immune cells from tissues. Current techniques, however, are limited in their ability to elucidate essential immune cell features, including variable sequences of T cell antigen receptors (TCRs) that confer antigen specificity. Here, we present a strategy that enables simultaneous analysis of TCR sequences and corresponding full transcriptomes from 3′-barcoded scRNA-seq samples. This approach is compatible with common 3′ scRNA-seq methods, and adaptable to processed samples post hoc. We applied the technique to identify transcriptional signatures associated with T cells sharing common TCRs from immunized mice and from patients with food allergy. We observed preferential phenotypes among subsets of expanded clonotypes, including type 2 helper CD4+ T cell (TH2) states associated with food allergy. These results demonstrate the utility of our method when studying diseases in which clonotype-driven responses are critical to understanding the underlying biology.

[Download paper here](http://duncanmorgan.github.io/files/wat3R.pdf)
